apps/api/src/app/events/usecases/message-matcher/message-matcher.usecase.ts

```diff
@@ -13,21 +13,31 @@ import {
   FilterPartTypeEnum,
   ICondition,
   TimeOperatorEnum,
+  ChannelTypeEnum,
+  IPreviousStepFilterPart,
+  PreviousStepTypeEnum,
 } from '@novu/shared';
 import { ExecutionDetailsSourceEnum, ExecutionDetailsStatusEnum } from '@novu/shared';
-import { SubscriberEntity, EnvironmentRepository, SubscriberRepository, StepFilter } from '@novu/dal';
+import {
+  SubscriberEntity,
+  EnvironmentRepository,
+  SubscriberRepository,
+  StepFilter,
+  ExecutionDetailsRepository,
+  MessageRepository,
+  JobRepository,
+  MessageEntity,
+} from '@novu/dal';
 
 import { IFilterVariables } from './types';
 import { FilterProcessingDetails } from './filter-processing-details';
-
-import {
-  CreateExecutionDetails,
-  CreateExecutionDetailsCommand,
-} from '../../../execution-details/usecases/create-execution-details';
+import { CreateExecutionDetails } from '../../../execution-details/usecases/create-execution-details';
 import { SendMessageCommand } from '../send-message/send-message.command';
 import { EXCEPTION_MESSAGE_ON_WEBHOOK_FILTER } from '../../../shared/constants';
-import { DetailEnum } from '../../../execution-details/types';
+import { CreateExecutionDetailsCommand } from '../../../execution-details/usecases/create-execution-details/create-execution-details.command';
```

Comment:
```suggestion
import { CreateExecutionDetailsCommand } from '../../../execution-details/usecases/create-execution-details';
```
It is available in the generic export. Less typing 🏌🏻